IPython/lib/deepreload.py

```diff
@@ -342,6 +342,19 @@ def reload(module, exclude=['sys', 'os.path', builtin_mod_name, '__main__']):
     finally:
         found_now = {}
 
+
+def _dreload(module, **kwargs):
+    """
+    **deprecated**
+
+    import reload explicitly from `IPython.lib.deepreload` to use it
+
+    """
+    warn("""
+injecting `dreload in interactive namespace is deprecated, please import `reload` explicitly from `IPython.lib.deeprelaod`
```

Comment:
[ni vu, ni connu, j't'embrouille](http://www.livinglanguage.com/community/discussion/286/idiomatic-expressions-and-slang/p1) (aka rebase-fixup-forcepush)